ARM-wvd-templates/HCI/QuickDeploy/AddHciVirtualMachinesQuickDeployTemplate.json

```diff
@@ -0,0 +1,269 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "hostPoolName": {
+        "type": "string",
+        "metadata": {
+          "description": "The name of the Hostpool to be created."
+        }
+      },
+      "hostpoolToken": {
+        "type": "securestring",
+        "metadata": {
+          "description": "The token of the host pool where the session hosts will be added."
+        }
+      },
+      "hostpoolResourceGroup": {
+        "type": "string",
+        "metadata": {
+          "description": "The resource group of the host pool to be updated. Used when the host pool was created empty."
+        }
+      },
+      "hostpoolLocation": {
+        "type": "string",
+        "metadata": {
+          "description": "The location of the host pool to be updated. Used when the host pool was created empty."
+        }
+      },
+      "hostpoolProperties": {
+        "type": "object",
+        "metadata": {
+          "description": "The properties of the Hostpool to be updated. Used when the host pool was created empty."
+        },
+        "defaultValue": {}
+      },
+      "vmAdministratorAccountUsername": {
+        "type": "string",
+        "metadata": {
+          "description": "A username to be used as the virtual machine administrator account."
+        }
+      },
+      "vmAdministratorAccountPassword": {
+        "type": "securestring",
+        "metadata": {
+          "description": "The password associated with the virtual machine administrator account."
+        }
+      },
+      "oUPath": {
+        "type": "string",
+        "metadata": {
+          "description": "OU Path for the domain join"
```

Comment:
Change to -
"Enter the name of the organizational unit (OU) you would like the virtual machines to join. If you do not have a specific OU, leave this parameter empty. The OU should look like OU=unit1,DC=contoso,DC=com [Learn more](https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names?WT.mc_id=Portal-Microsoft_Azure_WVD)"